### PR TITLE
Ss 4 change title for cart

### PR DESCRIPTION
**Pull request by socamb2**
@socamb Please have a look I think this is ready for QA 👍

### CODE REVIEW

File: SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -45,9 +45,9 @@
     <div>
         <b>Development Notes</b>
         <p>1.3.0.n - Team City DevOps</p>
-        <p>Sprint 1 will test out sprint development(1.3.1)</p>
-        <p>Work on SS-4 will be logged here</p>
-        <p>I see where to add comments</p>
+        <p>Sprint 1 will test our sprint development(1.3.1)</p>
+        <p>Work on SS-4 will be logged here. Add Comments here.</p>
+        <p>socamb2 - started working</p>
```
Comment (socamb) [inline SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml]: @socamb2 please note here that this is ready for QA Testing

File: SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -45,9 +45,9 @@
...
     </div>
```
Comment (socamb) [inline SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml]: What happens here

File: SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -45,9 +45,9 @@
     <div>
         <b>Development Notes</b>
         <p>1.3.0.n - Team City DevOps</p>
-        <p>Sprint 1 will test out sprint development(1.3.1)</p>
-        <p>Work on SS-4 will be logged here</p>
-        <p>I see where to add comments</p>
+        <p>Sprint 1 will test our sprint development(1.3.1)</p>
+        <p>Work on SS-4 will be logged here. Add Comments here.</p>
```
Comment (socamb2) [inline SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml:49]: @socamb how is this

File: SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -45,9 +45,9 @@
     <div>
         <b>Development Notes</b>
         <p>1.3.0.n - Team City DevOps</p>
-        <p>Sprint 1 will test out sprint development(1.3.1)</p>
```
Comment (socamb2) [inline SportsStore/SportsStore.WebUI/Views/Shared/_Layout.cshtml:48]: Out should be Our

**Review by socamb — APPROVED** [review]

**Pull request merged**